frame/support/test/tests/benchmark_ui/missing_origin.stderr

```diff
@@ -0,0 +1,5 @@
+error: Single-item extrinsic calls must specify their origin as the first argument.
+  --> tests/benchmark_ui/missing_origin.rs:11:3
+   |
+11 |         #[extrinsic_call]
```

Comment:
Sadly this isn't pointing to the extrinsic call, is it possible to do so?

Reply:
figured it out, the span I was using wasn't from the clone that actually gets used in the final output